### ISSUE TITLE
Postman v7.1.1 generate curl command with wrong content-length header when JSON request body has tabs

### ISSUE THREAD

**Issue by ninekaw9** [a85/POSTMan-Chrome-Extension#6607]
**Describe the bug**
Postman v7.1.1 generate wrong curl command content-length header when JSON request body has tabs

**To Reproduce**
1. Create a request that has JSON body with tabs indentation 
Example

```
{
	"AccountId":null,
	"Contact_Type__c":"Mobile App",
	"Contact_Channel__c":"",
	"Product_Type__c":"Traditional",
	"Activity_Type__c":"Mobile Application (S)",
	"Activity_Reason_1__c":"aaa (S)",
	"Activity_Reason_2__c":"aaa"
	,"Status":"Open",
	"Origin":"Mobile Application",
	"Priority":"Medium",
	"Description":"Proin cubilia congue rhoncus mi auctor porttitor vivamus mi faucibus etiam sit habitant platea."
}
```

```
curl -X POST \
  https://test.com/salesforce/case/v1 \
  -H 'Accept: */*' \
  -H 'Cache-Control: no-cache' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Postman-Token: 73c11ebc-f05b-4428-aebb-648ad77fd2ec,51b3d6db-a383-4b87-842a-6f497b734ea1' \
  -H 'User-Agent: PostmanRuntime/7.13.0' \
  -H 'accept-encoding: gzip, deflate' \
  -H 'cache-control: no-cache' \
  -H 'content-length: 412' \
  -H 'cookie: TS01f27908=0104f7a0ede114f64c7982d271251edf5ecadf19a8460efb0ba709d2c9cd701cf103f56bab6af9ec7dd179aa403a4ee8bc1ccd7167' \
  -H 'instance_url: https://cs6.salesforce.com' \
  -b TS01f27908=0104f7a0ede114f64c7982d271251edf5ecadf19a8460efb0ba709d2c9cd701cf103f56bab6af9ec7dd179aa403a4ee8bc1ccd7167 \
  -d '{
	"AccountId":null,
	"Contact_Type__c":"Mobile App",
	"Contact_Channel__c":"",
	"Product_Type__c":"Traditional",
	"Activity_Type__c":"Mobile Application (S)",
	"Activity_Reason_1__c":"aaa (S)",
	"Activity_Reason_2__c":"aaa"
	,"Status":"Open",
	"Origin":"Mobile Application",
	"Priority":"Medium",
	"Description":"Proin cubilia congue rhoncus mi auctor porttitor vivamus mi faucibus etiam sit habitant platea."
}'
```

2.Code -> curl
3.Run the curl command with -v, you will see that

> content-length: 412 (this is header generated from postman)
> instance_url: https://cs6.salesforce.com
>
> upload completely sent off: 401 out of 401 bytes  

**Expected behavior**
should be 
> content-length: 401

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.1.1
 - OS: macOS  10.14.5

**Comment by codenirvana** [comment]
@ninekaw9 Used the same curl snippet and got the correct content-length:
![Screenshot 2019-05-29 at 3 27 47 PM](https://user-images.githubusercontent.com/4865836/58548169-6c5e5380-8226-11e9-808d-56931eb90f09.png)

As a workaround, you can remove the auto-generated content-length header before sending the curl request. A permanent fix will be: #6472

Closing as duplicate of #6472

**Comment by ninekaw9** [comment]
![image](https://user-images.githubusercontent.com/9247289/58551375-08458a80-823a-11e9-8bb3-55c9b634c9eb.png)
just retested again, it is 401 bytes.
